src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs

```diff
@@ -31,6 +31,16 @@ public class Deluge : TorrentClientBase<DelugeSettings>
             _proxy = proxy;
         }
 
+        public override void MarkItemAsImported(DownloadClientItem downloadClientItem)
+        {
+            // set post-import category
+            if (!Settings.TvImportedCategory.Equals(Settings.TvCategory) &&
+                !Settings.TvImportedCategory.IsNullOrWhiteSpace())
```

Comment:
PS: we have a IsNotNullOrWhiteSpace.

You might want to swap the condition for readability. 
```
if (Settings.TvImportedCategory.IsNotNullOrWhiteSpace() &&
    Settings.TvImportedCategory != Settings.TvCategory)
```
Note that this changes the if to "is the function active AND is the category different", which is a bit more readable.